ir-client/README.md

```diff
@@ -0,0 +1,194 @@
+# Introduction
+
+This is an Image Registry client implementation compatible with an Image
+Registry server implementing [OCI
+Distribution Specification](https://github.com/opencontainers/distribution-spec)
+([pull workflow
+only](https://github.com/opencontainers/distribution-spec/blob/main/spec.md#pull))
+and optionally
+[Ra-TLS](https://github.com/islet-project/remote-attestation/tree/main/lib/ratls/ratls)
+protocol.
+
+# Purpose
+
+The main  purpose of this library is to provide a high abstraction
+client API to fetch an application image from OCI Registry server and
+unpack it to a given directory. This library also provides means to
+allow client verification using Ra-TLS protocol.
+
+This crate also provides a lower level OCI Registry client, which
+implements part of
+[OCI Distribution](https://github.com/opencontainers/distribution-spec/blob/main/spec.md#endpoints)
+client API. This allows to separately fetch manifest and blob files
+and list tags from the registry. This is not the main purpose of this
+library, thus it will not be described here.
+
+
+# Detailed description
+
+The main client provides below API:
+
+## Client initialization
+
+```
```

Comment:
Try to find some tag for the code block to enable syntax highlighting for rust. Not sure it's possible, but it should be.

Reply:
Done